AsyncDisplayKit/ASTextNode.mm

```diff
@@ -891,6 +891,10 @@ - (void)touchesMoved:(NSSet *)touches withEvent:(UIEvent *)event
 {
   [super touchesMoved:touches withEvent:event];
 
+  // on 3D Touch enabled phones, this gets fired with changes in force, and usually will get fired immediately after touchesBegan:withEvent:
+  if (CGPointEqualToPoint([[touches anyObject] previousLocationInView:self.view], [[touches anyObject] locationInView:self.view]))
```

Comment:
@tomizimobile please create a local variable for the touch, as well as a local variable for the location in view. Of course it is fine to avoid caching the previous location, since it is not used elsewhere.

This is important because if more than one touch is ever made available to this method, the set may return a different object when calling any object. Also, these methods are not free, and so it is worth the readability and performance advantages of having the local variables.